components/arm.py

```diff
@@ -47,6 +48,8 @@ class Arm:
     ROTATE_GRAVITY_FEEDFORWARDS = 2.5
 
     ARM_ENCODER_ANGLE_OFFSET = 0.325  # rotations 0-1
+    # time for the arm encoder to start working
+    ARM_STARTUP_TIME = 5
```

Comment:
5 seconds seems a bit pessimistic, how did we come up with this time? IIRC 3 seconds should be plenty of time for the RIO decoding to initialise.

Reply:
5 seconds was from my memory of the arm display when we deploy code, i think it was about 2-4 seconds before it started giving the correct value, i added a bit beacuse it freaking out in a match could cause serious damage and if the code is restarting an extra second or two wont matter much.

Reply:
Fair enough, thanks for the explanation.